### ISSUE TITLE
Group train/val/test metrics together

### ISSUE THREAD

**Issue by danielgafni** [dllllb/pytorch-lifestream#108]
Metrics should have names like `Train/loss`, `Train/recall_top_k`, `Train/<custom-metric>`,  `Val/loss`, `Test/loss`, ...

In this case Tensorboard (and other experiment tracking apps) will group `train` metrics together, which improves readability.

**Comment by ivkireev86** [comment]
Fixed in #120
